client/src/components/Header/components/universal-nav.css

```diff
@@ -25,15 +25,12 @@
   z-index: 2000;
 }
 
-.universal-nav-middle {
+#universal-nav-logo {
   display: flex;
   align-items: center;
-  flex: 1 0 33%;
-  overflow-x: auto;
-  overflow-y: hidden;
+  overflow: hidden;
+  max-inline-size: max-content;
```

Comment:
Flex and grid have a different way of calculating the width and height of SVG, for different browsers.

this aims to stop that behavior